cmd/controller/controller.go

```diff
@@ -300,7 +300,10 @@ func (c *command) start(ctx context.Context) error {
 		nodeComponents.Add(ctx, &applier.Manager{
 			K0sVars:           c.K0sVars,
 			KubeClientFactory: adminClientFactory,
-			LeaderElector:     leaderElector,
+			IgnoredStacks: map[string]string{
+				controller.ClusterConfigStackName: "v1.32",
```

Comment:
Maybe the `v1.32` should come from some constant

Reply:
We'd have to add a constant for each version where we add an ignored stack, and the value of that constant would never change. I don't see the immediate benefit. Can you explain this in more detail?

Reply:
Magic string. Where does it come from? Is there a matching "v1.32" string somewhere else in the code? Does someone have to change this string sometime? Why is it this single version that is ignored? Does or will v1.33 exist? Was v1.31 different? These are the questions that come to mind when seeing such string.

Reply:
I don't think `controller.ClusterConfigStackName` will change either? Maybe `controller.ClusterConfigStackVersion`?

Reply:
`controller.ClusterConfigStackName` is used in different parts of the code and has actual impact on k0s's behavior. This "v1.32" string is not really used at all in that sense. Maybe there's a misunderstanding in the meaning of  it? `controller.ClusterConfigStackVersion` wouldn't be appropriate, as "v1.32" is not referring the stack's version (it doesn't have any version in and of itself), it's referring to the first k0s version that ignored that stack folder. The only usage of "v1.32" is to be written into the "ignored.txt" file, to give admins a hint about the first k0s version that started to ignore this stack. We can also go without the values, and change the `IgnoredStacks` field from a map to a slice that only contains the stack names, and not put any version hint in that "ignored.txt" file.

> Where does it come from?

It's the k0s version in which k0s started to ignore the cluster config stack in the manifests applier.

> Is there a matching "v1.32" string somewhere else in the code?

No.

> Does someone have to change this string sometime?

Only if this PR doesn't make it into k0s 1.32, but into another k0s version.

> Why is it this single version that is ignored?

It's not a single version that's ignored, it's the k0s version at which k0s started to ignore a stack. It's not anticipated that this will change anymore.

> Does or will v1.33 exist?

Maybe, if we decide to ignore other stacks starting with k0s 1.33. However, this "v1.32" should remain unaffected, as we ignore this specific stack since version 1.32, which will not change.

> Was v1.31 different?

Yes, as we are relying on the cluster config stack to be applied by the manifest applier in that k0s version.

Reply:
In any case, I think here we see that some kind of clarification is necessary there :)

Reply:
I have removed the version hint. I couldn't think of a good way to express the intent without over-engineering :see_no_evil: (not that I'm not guilty of over-engineering from time to time :angel:).